packages/mobile/src/exchange/ExchangeTradeScreen.tsx

```diff
@@ -171,40 +183,62 @@ export class ExchangeTradeScreen extends React.Component<Props, State> {
     }
   }
 
+  // Output is either cGLD or cUSD based on input token
+  // Local amounts are converted to cUSD
+  getInputTokenAmount = () => {
+    const parsedInputAmount = parseInputAmount(this.state.inputAmount)
+    if (this.state.inputToken === CURRENCY_ENUM.GOLD) {
+      return parsedInputAmount
+    }
+
+    const { localCurrencyCode, localCurrencyExchangeRate } = this.props
+
+    let dollarsAmount
+    if (localCurrencyCode && localCurrencyCode !== LocalCurrencyCode.USD) {
```

Comment:
Rather than have multiple places in the code where we branch based on if local currency is not USD, could we just use the same code path but have the USD - USD exchange rate be 1?

Reply:
Yes this condition here is not really necessary, exchange rate is already set to 1 when USD currency is selected.